elements/lisk-cryptography/test/bls_lib/lib.spec.ts

```diff
@@ -86,6 +86,15 @@ describe('bls_lib', () => {
 				});
 			},
 		);
+
+		it('returns expected signature when secret key is zero', () => {
+			const sk = Buffer.alloc(32);
+			const message = Buffer.from('Hello, World!');
+
+			const signature = blsSign(sk, message);
+
+			expect(signature).toEqual(Buffer.concat([Buffer.from([192]), Buffer.alloc(95)]));
```

Comment:
This is tested in the fixture, so I think it's not needed?

Reply:
Ahh yes. I saw the comment on L74 `// Signing with the zero private key is not a use case according to the BLS specifications` and figured it's not been tested. But now I see the `/sign_case_zero_privkey/` is tested. Updated the test and pushing the changes.